cirq-core/cirq/qis/channels.py

```diff
@@ -50,11 +50,9 @@ def kraus_to_choi(kraus_operators: Sequence[np.ndarray]) -> np.ndarray:
         Choi matrix of the channel specified by kraus_operators.
     """
     d = np.prod(kraus_operators[0].shape, dtype=np.int64)
-    c = np.zeros((d, d), dtype=np.complex128)
-    for k in kraus_operators:
-        v = np.reshape(k, d)
-        c += np.outer(v, v.conj())
-    return c
+    k = np.asarray(kraus_operators)
+    flat_ops = k.reshape((-1, d))
```

Comment:
I think you can achieve the effect of `asarray` followed by `reshape` with `reshape` alone

```python
k = np.reshape(kraus_operators, (-1, d))
return np.einsum(..., k, k.conj())
```